docs/pages/eas-update/github-actions.mdx

```diff
@@ -161,3 +161,41 @@ Your GitHub Action should be set up now. Whenever a developer creates a pull req
 > Some repositories or organizations might need to explicitly enable GitHub Workflows and allow third-party Actions.
 
 </Step>
+
+## Using Bun instead of Yarn
+
+If you're using Bun as your package manager instead of Yarn, you'll need to make some modifications to the provided workflow. Below are the updated steps for both publishing updates on push and previews on pull requests:
```

Comment:
```suggestion
To use [Bun](/guides/using-bun/) as your package manager instead of Yarn, follow the steps below for both publishing updates on push and previews on pull requests:
```

Let's be a bit concise here.